File: src/components/info/augment/fusia.ts
import Augment, { AugmentGroup } from "./_base";
import { TRIPLE_OFF_EFF } from "../_effect";

const GROUP = "FUSIA";

let augments: Array<Augment> = [];

augments.push({
    name: "megas fusia",
    level: 0,
    effects: [
        ...TRIPLE_OFF_EFF.map((effect) => {
            return {
                effect,
                amount: 1.01,
            };
        }),
    ],
    battlepower: 4,
    group: GROUP,
});

const FUSIA: AugmentGroup = {
    name: GROUP,
    augments,
};

export default FUSIA;

File: _data/augments/fusia.py
from typing import List

from ._augment import Augment
from ._augment_groups import AugmentGroups
from effect import OFFENSIVE_POT
from util import many_effs_with_same_amount

GROUP = AugmentGroups.FUSIA
CONFLICT = (GROUP,)

augments: List[Augment] = []

# ----------------------------------------------------------------
augments.append(
    Augment(
        f"megas fusia",
        0,
        4,
        many_effs_with_same_amount(OFFENSIVE_POT, 1.01),
        GROUP,
        CONFLICT,
    )
)
# ----------------------------------------------------------------
